### PR TITLE
layers: Add Mesh DrawIndex undefined value

### PR DESCRIPTION
**Pull request by spencer-lunarg**
closes https://github.com/KhronosGroup/Vulkan-ValidationLayers/issues/7614

### CODE REVIEW

File: layers/core_checks/cc_pipeline_graphics.cpp [hunk 1]
```diff
@@ -1177,6 +1178,22 @@ bool CoreChecks::ValidateGraphicsPipelineBlendEnable(const vvl::Pipeline &pipeli
     return skip;
 }
 
+bool CoreChecks::ValidateGraphicsPipelineMeshTask(const vvl::Pipeline &pipeline, const Location &create_info_loc) const {
+    bool skip = false;
+    const bool has_mesh = (pipeline.active_shaders & VK_SHADER_STAGE_MESH_BIT_EXT) != 0;
+    const bool has_task = (pipeline.active_shaders & VK_SHADER_STAGE_TASK_BIT_EXT) != 0;
+    if (has_mesh && has_task) {
```
Comment (spencer-lunarg) [inline layers/core_checks/cc_pipeline_graphics.cpp:1185]: there are other mesh/task VUs I need to add here, they will need to know if there is a mesh and/or task so preparing for that

Comment (artem-lunarg) [reply layers/core_checks/cc_pipeline_graphics.cpp:1185]: > only the first active stage will have proper access to the variable

Does it mean we assume that task shader accessed draw index, so mesh shader can't use it?

Comment (spencer-lunarg) [reply layers/core_checks/cc_pipeline_graphics.cpp:1185]: hmm, thats a good question, let me look

Comment (spencer-lunarg) [reply layers/core_checks/cc_pipeline_graphics.cpp:1185]: confirmed the issue is when you have both a task and mesh stage in your pipeline, just having it in the mesh shader itself is illegal, there was no desire to let apps "just no access it" as built-ins are different than descriptors

**Review by artem-lunarg — APPROVED** [review]

**Pull request merged**